components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -61,6 +54,22 @@ export default class PostProfilePicture extends React.PureComponent {
         return this.props.status;
     };
 
+    getPostIconURL = (defaultURL, fromAutoResponder, fromWebhook) => {
+        const {enablePostIconOverride, hasImageProxy, post} = this.props;
+        const postIconOverrideURL = post.props.override_icon_url;
+        const useUserIcon = post.props.use_user_icon;
+
+        if (enablePostIconOverride) {
+            if (postIconOverrideURL && postIconOverrideURL !== '') {
```

Comment:
This logic appears to be subtly different, in that we now allow the override without checking `!fromAutoResponder && fromWebhook`. Can you elaborate on this requirement? I'm guessing it arises from the need to support the emoji, but at the same time allows a case for non-emoji that was previously excluded.

@hmhealey, do you happen to know why these constraints were previously imposed?

Reply:
Nice catch. The behaviour change was not intentional. I reverted and refactored, and will do some more testing.